examples/evaluate_benchmark_with_custom_api.py

```diff
@@ -0,0 +1,30 @@
+import unitxt
+from unitxt import evaluate, get_from_catalog, load_dataset
+from unitxt.text_utils import print_dict
+
+with unitxt.settings.context(
+    default_inference_api="watsonx",  # option a to define your home api
+    default_format="formats.chat_api",
+    disable_hf_datasets_cache=False,
+):
+    data = load_dataset("benchmarks.glue[max_samples_per_subset=5]", split="test")
+
+    model = get_from_catalog(
```

Comment:
I don't see why we need to use the catalog.

I think it's better to show this as:

MultiAPIInferenceEngine(model="llama_3_8b_instruct",api="watsonx",top_k=1)

It's more consistent with the other engines, and does not require non python  syntax.